lightning/webelement.py

```diff
@@ -0,0 +1,205 @@
+    def expression(self) -> str:
+        return self._expression
+
+    @property
+    def strategy(self) -> LocatorStrategy:
+        return self._strategy
+
+
+class By(Locator):
+
+    @staticmethod
+    def css_selector(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.CSS_SELECTOR)
+
+    @staticmethod
+    def xpath(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.XPATH)
+
+    @staticmethod
+    def tag_name(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.TAG_NAME)
+
+    @staticmethod
+    def link_text(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.LINK_TEXT)
+
+    @staticmethod
+    def partial_link_text(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.PARTIAL_LINK_TEXT)
+
+
+class Accessibility(dict):
+    def __init__(self, elements_instance: ElementsApi, session_id: str, id: str):
+        self._elements_instance = elements_instance
+        self._session_id = session_id
+        self._id = id
+
+    @property
+    def role(self) -> str:
+        _role = self._elements_instance\
+            .get_element_computed_role(session_id=self._session_id, element_id=self._id).value
+        return _role
+
+    @property
+    def label(self) -> str:
+        _label = self._elements_instance\
+            .get_element_computed_label(session_id=self._session_id, element_id=self._id).value
+        return _label
+
+    def __repr__(self):
+        _c = dict(role=self.role, label=self.label)
+        return repr(_c)
+
+
+class WebElement(dict):
+
+    def __init__(self, elements_instance: ElementsApi, id: str, session_id: str):
+        self._elements_instance = elements_instance
+        self._id = id
+        self._session_id = session_id
+
+    def accessibility(self) -> Accessibility:
+        _accessibility = Accessibility(elements_instance=self._elements_instance,
+                                       session_id=self.session_id,
+                                       id=self.id)
+        return _accessibility
+
+    @property
+    def id(self) -> str:
+        return self._id
+
+    @property
+    def session_id(self) -> str:
+        return self._session_id
+
+    def click(self) -> WebElement:
+        _web_element = self._elements_instance.element_click(session_id=self.session_id,
+                                                             element_id=self.id,
+                                                             body=EmptyRequest())
+        return self
+
+    def clear(self) -> WebElement:
+        try:
+            _web_element = self._elements_instance.element_clear(session_id=self.session_id,
+                                                                 element_id=self.id,
+                                                                 body=EmptyRequest())
+
+            return self
+        except ApiException as e:
```

Comment:
You need to apply the same to all methods, not only this one. To not copy-paste a lot - use decorator for that: https://stackoverflow.com/questions/15572288/general-decorator-to-wrap-try-except-in-python